reference/enterprise-api-ref/ssh-keys-api.markdown

```diff
@@ -0,0 +1,151 @@
+---
+layout: default title: SSH keys API published: true tags: [reference, enterprise, API, build, SSH]
+---
+
+The SSH keys API enables you to generate a key pair that can be used for authorization.
+
+# SSH keys API
+
+## Generate SSH key
+
+**URI:** https://hub.cfengine.com/api/ssh-key
```

Comment:
you could add similar details as above, "generates a key with 4096 bits, sha2-512 digest and in rfc4716 (default openssh) format" or similar.

Reply:
done